content/blog/2023-11-13-announcing-localstack-30-general-availability/index.md

```diff
@@ -0,0 +1,186 @@
+
+- **500 Internal Error**: This experiment randomly terminates incoming requests and returns an `Internal error` with a response code of 500.
+- **Service Unavailable**: Similar to the previous one, this experiment randomly terminates incoming requests but responds with a `Service Unavailable` message and a 503 response code.
+- **AWS Region Unavailable**: This simulates region outages and failovers by randomly terminating incoming requests to mimic regional service disruptions.
+- **Latency**: This introduces a specified amount of latency to each API call, useful for simulating slow network conditions or network performance issues.
+
+{{< img-simple src="localstack-chaos-engineering-dashboard.png" width=300 alt="Image of LocalStack Chaos Engineering dashboard">}}
+
+Additionally, we have introduced user guides for various scenarios, including simulating unexpected outages using the [LocalStack Outage Extension](), implementing [Route53 Failover with FIS](), and configuring error probabilities in [Kinesis]() & [DynamoDB]().
+
+These resources are designed to help users effectively respond to such scenarios, facilitating the development of thorough disaster recovery plans. By using these tools, teams can prepare to sustain stability and efficiency even in challenging situations. Check out our [documentation]() for more information.
+
+### IAM Policy Stream on Web Application
+
+In application development, accessing AWS resources like S3 buckets and RDS databases is common. To grant access, we create IAM roles and attach policies that specify permissions. However, determining the correct permissions can be challenging, often leading developers to assign excessive permissions to IAM roles. To address this, LocalStack introduced the **IAM Policy Stream**. This tool simplifies identifying the necessary permissions for cloud applications and helps detect logical errors.
+
+We have expanded the IAM Policy Stream feature to be accessible directly through the Web Application, in addition to the existing CLI feature. This enhancement will display the specific policy generated for each API call in the new interface, simplifying permission management and eliminating concerns about assigning correct permissions.
+
+The features include:
+
+1.  A real-time list of calls and the corresponding policies they generate.
+2.  A real-time summary policy that combines all individual policies into a single comprehensive policy.
+3.  The option to enable or disable this feature during runtime, allowing for performance optimization as needed.
+4.  The ability to reset the stream, enabling a fresh start with a new set of policies.
+
+{{< img-simple src="localstack-IAM-policy-stream.png" width=300 alt="Image of LocalStack IAM Policy Stream dashboard">}}
+
+Check out our [documentation]() and [video]() on getting started with the IAM Policy Stream.
+
+### Ephemeral Environments for LocalStack
```

Comment:
add (coming soon) to header

Reply:
+1 - either "coming soon" or "Preview" or "Beta"